### DISCUSSION TITLE
wire:ignore.self doesn't work

### DISCUSSION THREAD

**Discussion by N1ebieski** [livewire/livewire#7663] [7. Bugs (archived)]
### Livewire version

v3.3.4

### Laravel version

v10.0.0

### Which PHP version are you using?

PHP 8.3

### Steps To Reproduce

1. Go to my wirebox > https://wirebox.app/b/yrdv2
2. Open the console in the browser (F12)
3. When rendering the page, messages "It should show only once!" will appear
4. Click the button "Toggle order", at least 3 new messages will appear in the console

Messages shouldn't appear again because the div with alpine component has an attribute: ```wire:ignore.self```.

This bug only occurs when any children element inside div with ```wire:ignore.self``` has some input or button.

This bug didn't occur in version 3.3.0 > https://wirebox.app/b/4n5mj

### Repository or Wirebox URL

https://wirebox.app/b/yrdv2

### Additional information

Guys, I know you have much to do but please, stop releasing new features and fix the bugs first. Livewire is bugged a lot, I've personally reported at least a few that have not been fixed since the release of version 3.0.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by N1ebieski** [comment]
Still not fixed in the version 3.4.2 > https://wirebox.app/b/41dgw

**Comment by calebporzio** [comment]
Can you submit a PR with a falling test?

**Reply by N1ebieski** [reply]
Unfortunately, I cant configure Dusk in my docker environment

**Reply by calebporzio** [reply]
Mmm ok, can you still submit a failing test? Even if you can't run it, it will be helpful for us
